packages/flutter/lib/src/material/app.dart

```diff
@@ -76,20 +122,20 @@ class MaterialApp extends StatefulWidget {
   /// The colors to use for the application's widgets.
   final ThemeData theme;
 
-  /// The widget for the default route of the app
-  /// ([Navigator.defaultRouteName], which is `'/'`).
+  /// The widget for the default route of the app ([Navigator.defaultRouteName],
+  /// which is `/`).
   ///
-  /// This is the page that is displayed first when the application is
-  /// started normally.
+  /// This is the page that is displayed first when the application is started
```

Comment:
Because routes are pages on the Web and old habits die hard.

Changed the two "page"s to "route"s.